SabreTools.Models/PlayStation4/AppPkgHeader.cs

```diff
@@ -0,0 +1,238 @@
+using System.Runtime.InteropServices;
+
+namespace SabreTools.Models.PlayStation4
+{
+    /// <see href="https://www.psdevwiki.com/ps4/PKG_files"/>
+    [StructLayout(LayoutKind.Sequential)]
+    public class AppPkgHeader
+    {
+        /// <summary>
+        /// 0x7F434E54
+        /// ".CNT"
+        /// </summary>
+        [MarshalAs(UnmanagedType.ByValArray, SizeConst = 4)]
+        public byte[]? Magic;
+
+        /// <summary>
+        /// PKG Type
+        /// </summary>
+        public uint Type;
+
+        /// <summary>
+        /// PKG Unknown Field
+        /// </summary>
+        public uint PKGUnknown;
+
+        /// <summary>
+        /// PKG File count
+        /// </summary>
+        public uint FileCount;
+
+        /// <summary>
+        /// PKG Entry count
+        /// </summary>
+        public uint EntryCount;
+
+        /// <summary>
+        /// SC Entry count
+        /// </summary>
+        public ushort SCEntryCount;
+
+        /// <summary>
+        /// PKG Entry count (duplicated)
+        /// </summary>
+        public ushort EntryCount2;
+
+        /// <summary>
+        /// PKG Table offset
+        /// </summary>
+        public uint TableOffset;
+
+        /// <summary>
+        /// PKG Entry data size
+        /// </summary>
+        public uint EntryDataSize;
+
+        /// <summary>
+        /// PKG Body offset
+        /// </summary>
+        public ulong BodyOffset;
+
+        /// <summary>
+        /// PKG Body size
+        /// </summary>
+        public ulong BodySize;
+
+        /// <summary>
+        /// PKG Content offset
+        /// </summary>
+        public ulong ContentOffset;
+
+        /// <summary>
+        /// PKG Content size
+        /// </summary>
+        public ulong ContentSize;
+
+        /// <summary>
+        /// PKG Content ID
+        /// </summary>
+        [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 36)]

```

Comment:
Here and other places, it's okay to use hex values in the `SizeConst` field, especially if it lines up better with the original documentation.